src/components/VerticalCard/index.jsx

```diff
@@ -149,7 +154,7 @@ const VerticalCard = ({ product }) => {
           </p>
         </div>
       )}
-      <div className="zoom-wrapper">
+      <div className="zoom-wrapper" onClick={navigateToDetails}>
```

Comment:
Suggestion : instead of having a separate function you can also use the `<Link>` tag here like you did in line 212.

Reply:
Thanks for the suggestion, actually I did not want to wrap an image with a link tag.